backend/dal/Services/Concrete/NotificationQueueService.cs

```diff
@@ -308,7 +308,14 @@ public NotificationQueue GenerateNotification(Project project, NotificationTempl
 
             var env = new EnvironmentModel(_options.Environment.Uri, _options.Environment.Name, _options.Environment.Title);
             var model = new ProjectNotificationModel(Guid.NewGuid(), env, project, project.Agency);
-            return GenerateNotification(project.Agency.Email, template, model);
+            var email = project.Agency.Email;
```

Comment:
Ideally don't make a request unless the email is null.
`var email = !String.IsNullOrWhiteSpace(project.Agency.Email) ? project.Agency.Email : this.Context.Users.FirstOrDefault(u => u.Id == project.CreatedById).Email;`